server/server.go

```diff
@@ -540,6 +542,23 @@ func (s *Server) GetLabelProperty() LabelPropertyConfig {
 	return s.scheduleOpt.loadLabelPropertyConfig().clone()
 }
 
+// SetClusterVersion sets the version of cluster.
+func (s *Server) SetClusterVersion(v string) error {
+	version, err := ParseVersion(v)
+	if err != nil {
+		return errors.Trace(err)
+	}
+	s.scheduleOpt.SetClusterVersion(*version)
+	s.scheduleOpt.persist(s.kv)
```

Comment:
Unchecked return value?

Reply:
There are at least 4 other instances of not checking the return value for this in the same file. I am going to try to setup errcheck efficiently, but I may not finish that until next week. But here is a simple check of the server code: https://github.com/pingcap/pd/pull/1152